OctaneSdk/Entities/Base/BaseEntity.cs

```diff
@@ -157,5 +157,14 @@ public override string ToString()
             String name = Name == null ? "No name" : Name.Substring(0, Math.Min(20, Name.Length)); //Show upto 20 characters in name
             return m_properties == null ? "No properties" : String.Format("{0} #{1} - {2} , {3} properties", TypeName, Id, name, m_properties.Count);
         }
+
+        public BaseEntity SimpleClone()
```

Comment:
Its very vague name for Clone method. 
Implementation also is too custom.
Please think of something generic.

Reply:
Removed clone method as I realized it was no longer needed